Formula/servus.rb

```diff
@@ -0,0 +1,30 @@
+class Servus < Formula
+  desc "Library and Utilities for zeroconf networking"
+  homepage "https://github.com/HBPVIS/Servus"
+  url "https://github.com/HBPVIS/Servus/archive/1.5.0.tar.gz"
+  sha256 "42fb9c060f17f040ad3c7563f5e87c89f5a221a5aa7da21384fc26b9c725ecc8"
+
+  bottle do
+    cellar :any
+    sha256 "fb13c90a02d860b5442a046a66a98f190788b2d5f576c574b891751a14353893" => :el_capitan
+  end
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+
+    # Install tests to share/Servus/tests so that we can test later
```

Comment:
Would rather use an integration test instead i.e. actually using the library and utilities.